.github/workflows/ci-cd.yaml

```
@@ -43,13 +43,17 @@ jobs:
   deploy-test:
     name: Internal test
     runs-on: ubuntu-latest
-    needs: [get-version, publish]
+    environment: test
     if: always() && !failure() && !cancelled() 
```

yamllint

[error] trailing spaces

(trailing-spaces)